package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nzz/q-server",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
lets go with 2.1.1 here. There is no new feature in this release. I'd consider the missing `reduce: false` from 2.1.0 a bug.